### ISSUE TITLE
npm login w/ custom cert + key is broken in npm@5.5

### ISSUE THREAD

**Issue by wejendorp** [npm/npm#19041]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [x] Other (_see below for feature requests_):

#### What's going wrong?
`npm login` doesn't use the provided cert and key, causing `EPROTO` errors on custom registry with mutual TLS.

Works in `npm@5.3` and `npm@5.4`. In 5.5 I get:
```
npm ERR! code EPROTO
npm ERR! errno EPROTO
npm ERR! request to https://${registry}/-/user/org.couchdb.user:${myuser} failed, reason: write EPROTO 140737250317248:error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure:../deps/openssl/openssl/ssl/s3_pkt.c:1494:SSL alert number 40
npm ERR! 140737250317248:error:1409E0E5:SSL routines:ssl3_write_bytes:ssl handshake failure:../deps/openssl/openssl/ssl/s3_pkt.c:659
```

Other npm requests work fine w/ the certs, just login is failing.

#### How can the CLI team reproduce the problem?
```
; ~/.npmrc
cert = "..."
key = "..."
```

`npm login`

### supporting information:
```
$ npm -v
5.5.1
$ node -v
8.9.0
```

**Comment by wejendorp** [comment]
Fix posted as #19048

**Comment by wejendorp** [comment]
Closing this issue, fix was released in `npm@5.6.0`.
